src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -365,17 +365,17 @@ public virtual void WriteLiteralTo(TextWriter writer, string value)
         }
 
         public virtual void WriteAttribute(string name,
```

Comment:
nit: Since you're touching this if you could just move all params as single tabbed instead of being X tab spaced (easier to maintain), ditto for other `WriteAttributeTo`